pkg/controller/jobs/kubeflow/jobs/pytorchjob/pytorchjob_controller_test.go

```diff
@@ -167,3 +168,50 @@ func TestCalcPriorityClassName(t *testing.T) {
 		})
 	}
 }
+
+func TestOrderedReplicaTypes(t *testing.T) {
```

Comment:
this looks outdated

Reply:
Why do you think this is outdated? I think this test works fine.